pkg/process/checks/process_common_test.go

```diff
@@ -21,6 +21,19 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/process/procutil"
 )
 
+// chunking and no chunking options used for testing chunking of check output
+var (
+	chunkingOptions = &RunOptions{RunStandard: true, NoChunking: false}
+	noChunkingOptions = &RunOptions{RunStandard: true, NoChunking: true}
+)
```

Comment:
I don't think we should have global vars for this. Maybe instead a helper function that returns what you need?